client/src/components/main/RsvpListModal.tsx

```diff
@@ -92,27 +71,27 @@ export default function RsvpListModal() {
           <TableHeader>
             <TableRow>
               <TableHead className="border-b border-r border-[#7D916F] text-black">
-                First Name
+                Username
               </TableHead>
               <TableHead className="border-b border-r border-[#7D916F] text-black">
                 Last Name
               </TableHead>
-              <TableHead className="border-b border-[#7D916F] text-black">
+              <TableHead className="border-b border-r border-[#7D916F] text-black">
                 Email
               </TableHead>
             </TableRow>
           </TableHeader>
           <TableBody className="text-black">
-            {attendees.map((a) => (
-              <TableRow key={a.email}>
+            {attendees?.map((a) => (
+              <TableRow key={a.user.email}>
                 <TableCell className="border-r border-t border-r-[#7D916F] border-t-[#DEE4DB]">
-                  {a.firstName}
+                  {a.user.first_name}
                 </TableCell>
                 <TableCell className="border-r border-t border-r-[#7D916F] border-t-[#DEE4DB]">
-                  {a.lastName}
+                  {a.user.last_name}
                 </TableCell>
-                <TableCell className="border-t border-t-[#DEE4DB] text-[#5C764B] underline">
-                  <a href={`mailto:${a.email}`}>{a.email}</a>
+                <TableCell className="text6789-[#5C764B] border-t border-t-[#DEE4DB] underline">
```

Comment:
is there a typo here?

Reply:
I'm not sure, since we only connected the modal to the backend. We did not write up the modal.

Reply:
Yeah not sure what that is. The numbers '6789' can be removed.